### PR TITLE
Add app.drey.Damask

### PR DESCRIPTION
**Pull request by subpop**
### Please confirm your submission meets all the criteria

- [x] I have read the [App Requirements][reqs] and [App Maintenance][maint] pages.
- [x] My pull request follows the instructions at [App Submission][submission].
- [x] I am using only the minimal set of permissions. *(If not, please explain each non-standard permission.)*
- [x] All assets referenced in the manifest are redistributable by any party.  If not, the unredistributable parts are using an extra-data source type.
- [x] I am an upstream contributor to the project. If not, I contacted upstream developers about submitting their software to Flathub. **Link:**
- [x] I own the domain used in the application ID or the domain has a policy for delegating subdomains (e.g. GitHub, SourceForge).
- [x] Any additional patches or files have been submitted to the upstream projects concerned. *(If not, explain why.)*

[reqs]: https://github.com/flathub/flathub/wiki/App-Requirements
[maint]: https://github.com/flathub/flathub/wiki/App-Maintenance
[submission]: https://github.com/flathub/flathub/wiki/App-Submission

### CODE REVIEW

File: app.drey.Damask.json [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+        "--share=ipc",
+        "--socket=fallback-x11",
+        "--device=dri",
+        "--socket=wayland"
+    ],
+    "build-options": {
+        "append-path": "/usr/lib/sdk/vala/bin",
+        "prepend-ld-library-path": "/usr/lib/sdk/vala/lib",
+        "env": {}
+    },
```
Comment (hfiguiere) [inline app.drey.Damask.json:18]:
```suggestion
```

you do not need these.

File: app.drey.Damask.json [hunk 2]
```diff
@@ -0,0 +1,72 @@
...
+        "/share/pkgconfig",
+        "/share/vala",
+        "*.la",
+        "*.a"
+    ],
+    "modules": [
+        {
+            "name": "blueprint-compiler",
+            "buildsystem": "meson",
+            "sources": [
```
Comment (hfiguiere) [inline app.drey.Damask.json:35]: there should be a cleanup '*' rule do remove the entirety of the blueprint build.

**Comment by hfiguiere** [comment]
bot, build app.drey.Damask

File: app.drey.Damask.json (hunk 1 above)
Comment (subpop) [reply app.drey.Damask.json:18]: Oh interesting. They came from the manifest template that Builder uses for new projects.

File: app.drey.Damask.json (hunk 2 above)
Comment (subpop) [reply app.drey.Damask.json:35]: Ahh yes of course. Thank you!

**Comment by subpop** [comment]
bot, build app.drey.Damask

**Pull request closed without merging**